### PR TITLE
QPID-8515:[Broker-J]Add missing operational logs

### PR DESCRIPTION
**Pull request by deepyaraj**
Every change to the broker configuration has to be logged, few update scenarios and other create and delete weren't logged. Implemented operational logging for all the missing ones.

### CODE REVIEW

File: broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java [hunk 1]
```diff
@@ -2974,6 +2979,79 @@ else if(desiredState instanceof String)
...
+                                                               getType(),
+                                                               getName(),
+                                                               attributesString));
+        }
+    }
+
+    protected void createOperationalLog(final Map<String, Object> attributes){
+        final String attributeString = attributes.entrySet().stream()
+                                                 .map(e -> {
+                                                     ConfiguredObjectAttribute attributeType =
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java:3016]: duplicate code

File: broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java [hunk 2]
```diff
@@ -722,7 +728,7 @@ public void onFailure(final Throwable t)
                 childCloseFutures.add(childCloseFuture);
             }
         });
-
+        //deleteOperationalLog();
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java:731]: commented code

File: broker-core/src/main/java/org/apache/qpid/server/logging/messages/Management_logmessages.properties [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# 0 - category
+# 1 - type
+# 2 - name
+DELETE = MNG-2003 : Deleted : {0} : {1} : {2}
+
+# 0 - category
+# 1 - type
+# 2 - name
+# 3 - parameters
+OPERATION = MNG-2003 : Deleted : {0} : {1} : {2}  {3}
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/logging/messages/Management_logmessages.properties:43]: new line expected

File: broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java [hunk 4]
```diff
@@ -313,6 +313,7 @@ public String getPath()
         public GroupAdapter(Map<String, Object> attributes)
         {
             super(FileBasedGroupProviderImpl.this, attributes);
+            createOperationalLog(attributes);
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java:316]: let's move call into onCreate

File: broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java [hunk 5]
```diff
@@ -391,6 +392,7 @@ public GroupMemberAdapter(Map<String, Object> attrMap)
             {
                 // TODO - need to relate to the User object
                 super(GroupAdapter.this, attrMap);
+                createOperationalLog(attrMap);
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java:395]: let's move call into onCreate

File: broker-core/src/main/java/org/apache/qpid/server/security/group/GroupImpl.java [hunk 6]
```diff
@@ -44,6 +44,7 @@ public GroupImpl(Map<String, Object> attributes,
                      GroupProvider<?> provider)
     {
         super(provider, attributes);
+        createOperationalLog(attributes);
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/security/group/GroupImpl.java:47]: let's move call into onCreate

File: broker-core/src/main/java/org/apache/qpid/server/security/group/GroupMemberImpl.java [hunk 7]
```diff
@@ -43,6 +43,7 @@
     public GroupMemberImpl(Map<String, Object> attributes, Group group)
     {
         super((ConfiguredObject<?>) group, attributes);
+        createOperationalLog(attributes);
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/security/group/GroupMemberImpl.java:46]: let's move call into onCreate

**Review by alex-rufous — CHANGES_REQUESTED** [review]

File: broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java (hunk 2 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/model/AbstractConfiguredObject.java:731]: Removed !

File: broker-core/src/main/java/org/apache/qpid/server/logging/messages/Management_logmessages.properties (hunk 3 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/logging/messages/Management_logmessages.properties:43]: Added!

File: broker-core/src/main/java/org/apache/qpid/server/security/group/GroupImpl.java (hunk 6 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/security/group/GroupImpl.java:47]: Done!

File: broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java (hunk 5 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java:395]: Done!

File: broker-core/src/main/java/org/apache/qpid/server/security/group/GroupMemberImpl.java (hunk 7 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/security/group/GroupMemberImpl.java:46]: Done!

File: broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java (hunk 4 above)
Comment (deepyaraj) [reply broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java:316]: Done!

File: broker-core/src/main/java/org/apache/qpid/server/queue/QueueConsumerImpl.java [hunk 8]
```diff
@@ -568,7 +568,7 @@ public String toLogString()
         }
         else
         {
-            String queueString = new QueueLogSubject(_queue).toLogString();
+            String queueString = new QueueLogSubject(getName(), getName()).toLogString();
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/queue/QueueConsumerImpl.java:571]: the second argument should be a virtual host name

**Review by alex-rufous — CHANGES_REQUESTED** [review]

**Pull request closed without merging**